mdx/main.py

```diff
@@ -668,9 +675,22 @@ def process_mdx_cumulus(self):
         [cumulus_granules_meta.pop(ele, False) for ele in ['granuleId', 'files']]
         self.input = []
         for granule in granules:
-            for _file in granule['files']:
-                if 'metadata' not in _file.get('type', ""):
-                    self.input.append(f"s3://{_file['bucket']}/{_file['key']}")
+            files = granule['files']
+            i = 0
+            while i < len(files):
+                file = files[i]
+                if file.get('fileName', '').endswith('.cmr.json'):
+                    logger.info(f'Removing old metadata file: {file.get("fileName", "")}')
+                    files.pop(i)
```

Comment:
Is this just to avoid processing the metadata file as an input? How does this look within the Cumulus database? i.e. is there now two metadata files associate with this one input or will the newly created metadata file overwrite the existing one in the Cumulus database as well?

Reply:
Side effects on the Cumulus db not tested.

Reply:
Mike confirmed that updating the granule this way will result in only 1 file entry for the `.cmr.json` file within the files table of the Cumulus db. 👍